web/src/app/admin/admin-feedback-section.component.scss

```
@@ -0,0 +1,44 @@
+  font-family: monospace;
+  font-size: 0.85em;
+  background: var(--mat-sys-surface-variant);
+  padding: 2px 6px;
+  border-radius: 4px;
+}
+mat-table {
+  width: 100%;
+}
+.message-cell {
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)